packages/chains/src/polygon/redemptionStrategies.ts

```diff
@@ -152,6 +152,8 @@ const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, stri
     underlying(assets, assetSymbols.USDC),
   ],
   [underlying(assets, assetSymbols["MAI-USDC"])]: [RedemptionStrategyContract.UniswapLpTokenLiquidator, USDC],
+  [underlying(assets, assetSymbols["WMATIC-MATICx"])]: [RedemptionStrategyContract.UniswapLpTokenLiquidator, WMATIC],
+  [underlying(assets, assetSymbols.MIMO_PAR_80_20)]: [RedemptionStrategyContract.BalancerLpTokenLiquidator, PAR],
```

Comment:
@vminkov review here: using PAR as output, how would we get USDC back instead?

Reply:
We need a redemption strategy for that, check the on-chain liquidity for such sources

Reply:
Sadly there's not much, except.... 

jEUR-PAR curve: https://polygon.curve.fi/factory/285

Then redeem jEUR for USDC. Wdyt?

Reply:
https://github.com/Midas-Protocol/monorepo/pull/1185/commits/7f922552b60c8c0f15447a2395b43738edd35074

Not entirely sure this is correct